octodns_azure/__init__.py

```diff
@@ -42,7 +42,7 @@
 from octodns.record import GeoCodes, Record, Update
 
 # TODO: remove __VERSION__ with the next major version release
-__version__ = __VERSION__ = '0.0.7'
+__version__ = __VERSION__ = '0.0.8'
```

Comment:
Please revert this change. We always do release bumps & changelog updates in a dedicated release PR. 

```suggestion
__version__ = __VERSION__ = '0.0.8'
```

Reply:
reverted! :)